tee-worker/ts-tests/integration-tests/scheduled_enclave.test.ts

```diff
@@ -38,47 +38,4 @@ describe('Scheduled Enclave', function () {
         assert.equal(blockNumber, 0);
         assert.equal(mrEnclave, context.mrEnclave);
     });
-
```

Comment:
Sadly I had to remove this step. This is `development` feature so it's not possible to test it with worker built without it.

Reply:
sounds reasonable 👍  It's me who implemented this test in a wrong way. I had to use indirect invocation instead

Reply:
I've prepared PR to use II for setting enclave instead of using dev RPC method
https://github.com/litentry/litentry-parachain/pull/2664